pkgs/applications/virtualization/xen/generic/0000-xen-ipxe-src-generic.patch

```diff
@@ -1,6 +1,6 @@
 Hack to make etherboot use pre-fetched iPXE.
 
-diff --git a/tools/firmware/etherboot/Makefile b/tools/firmware/etherboot/Makefile
+diff a/tools/firmware/etherboot/Makefile b/tools/firmware/etherboot/Makefile
```

Comment:
What happened here? Looks like a Git format diff.

Reply:
Just cosmetics. The flag doesn't really matter one way or the other, so I thought I'd just keep the path. Another patch by Lach had other flags that I also removed for consistency, so I standardised this one as well.